tests/unit/email.spec.ts

```diff
@@ -1,30 +1,36 @@
 import Email from "@/utils/email";
 import { ValueError, ValidationError } from "@/errors";
 
-test("Email is requiring value", () => {
-  try {
-    expect(new Email()).toThrow(ValueError);
-  } catch (e) {
-    expect(e).toBeInstanceOf(ValueError);
-  };
-});
+describe("Email", () => {
+  it("can be set", () => {
+    expect(new Email("firstname@domain.com")).toBeInstanceOf(Email);
+  });
 
-test("Email length check", () => {
-  try {
-    expect(new Email("")).toThrow(ValueError);
-  } catch (e) {
-    expect(e).toBeInstanceOf(ValueError);
-  };
-});
+  it("can be converted to string", () => {
+    expect(new Email("firstname@domain.com").toString()).toBe("firstname@domain.com");
+  });
 
-test("Email validation check", () => {
-  try {
-    expect(new Email("a")).toThrow(ValidationError);
-  } catch (e) {
-    expect(e).toBeInstanceOf(ValidationError);
-  };
-})
+  it("requiring value", () => {
```

Comment:
I wouldn't phrase it that way. These comments for the `it` statements should be written as English sentences. So it might be changed to `"requires a value"`